src/Illuminate/Database/DatabaseServiceProvider.php

```diff
@@ -74,8 +74,8 @@ protected function registerConnectionServices()
      */
     protected function registerEloquentFactory()
     {
-        $this->app->singleton(FakerGenerator::class, function ($app) {
-            return FakerFactory::create($app['config']->get('app.faker_locale', 'en_US'));
+        $this->app->singleton(FakerGenerator::class, function ($app, array $params) {
```

Comment:
Should probably make this params argument optional?

Reply:
Laravel versions 5.0+ send an array as the second parameter to the callback. Besides that, this change is not meant to be backwards compatible with the previous version of Laravel.